src/generators/ts/ts.generator.ts

```diff
@@ -106,20 +106,27 @@ export class TsGenerator extends BuildFileGenerator {
     let label = this.workspace.tryResolveLabelFromStaticMapping(imp, undefined, '.');
     if (label) { return label; }
 
-    // some imports are deep but resolve to one label (eg material and cdk)
-    if (imp.startsWith('@angular/material')) {
-      return Label.parseAbsolute(`@${npmWorkspace}//@angular/material`);
-    } else if (imp.startsWith('@angular/cdk')) {
-      return Label.parseAbsolute(`@${npmWorkspace}//@angular/material`);
-    }
+    const relative = this.workspace.isWorkspaceRelative(imp) || imp.startsWith('.');
 
-    if (imp.startsWith('.')) {
+    if (relative) {
       label = this.workspace.getLabelForFile(imp + '.ts');
       if (label) { return label; }
-    }
 
-    // fall back to assuming 3rd_party
-    return Label.parseAbsolute(`@${npmWorkspace}//${imp}`);
+      throw new Error (`Unable to generate label for: ${imp}`)
+    } else {
+      const pathParts = imp.split(sep);
```

Comment:
Won't the path always be separated with `/` regardless of the platform? `imp` holds the path in the module specifier.

Reply:
ah yes, this would have broken on windows otherwise